runFIT.sh

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash -ex
+
+#############################################
+#
+# Usage
+############################################
+Usage(){
+    set +x
+    echo "Function: This script is used to set up environment for FIT and run FIT."
+    echo "Usage: $0 [OPTIONS]"
+    echo "  OPTIONS:"
+    echo "    Mandatory options:"
+    echo "      -w, --WORKSPACE: The directory of workspace( where the code will be cloned to and staging folder), it's required"
+    echo "      -p, --SUDO_PASSWORD: password of current user which has sudo privilege, it's required."
+    echo "    Optional options:"
+    echo "      -g, --TEST_GROUP: test group of FIT, such as imageservice, smoke"
+    echo "      -r, --RACKHD_DIR: The directory of repository RackHD"
+    echo "                       If it's not provided, the script will clone the latest repository RackHD under $WORKSPACE"
+    set -x
+}
+
+
+#############################################
+#
+#  Create the virtual env for FIT  
+#
+############################################
+setupVirtualEnv(){
+    pushd ${RACKHD_DIR}/test
+    virtual_env_name=FIT
+    rm -rf .venv/$virtual_env_name
+    ./mkenv.sh $virtual_env_name
+    source myenv_$virtual_env_name
+    popd
+}
+
+####################################
+#
+# 1. Modify FIT config files , to  using actual DHCP Host IP instead of 172.31.128.1
+#
+##################################
+setupTestsConfig(){
+    echo "SetupTestsConfig ...replace the 172.31.128.1 IP in test configs with actual DHCP port IP"
+    RACKHD_DHCP_HOST_IP=$(ifconfig | awk '/inet addr/{print substr($2,6)}' |grep 172.31.128)
+    if [ "$RACKHD_DHCP_HOST_IP" == "" ]; then
+         echo "[Error] There should be a NIC with 172.31.128.xxx IP in your OS."
+         exit -2
+    fi
+    pushd ${RACKHD_DIR}/test/config
+    sed -i "s/\"username\": \"vagrant\"/\"username\": \"${USER}\"/g" credentials_default.json
+    sed -i "s/\"password\": \"vagrant\"/\"password\": \"$SUDO_PASSWORD\"/g" credentials_default.json
+    popd
+    pushd ${RACKHD_DIR}/test
+    find ./ -type f -exec sed -i -e "s/172.31.128.1/${RACKHD_DHCP_HOST_IP}/g" {} \;
```

Comment:
Erika will hate this ...... aha ...
we will eventually find the exact place about the IP in the future( Not this PR)